lg.pg.admin.php

```diff
@@ -23,12 +23,13 @@
 define('lg_home_admin_licnotvalid','Lizenz nicht gültig. Bitte überprüfen ob die aktuellste Lizenz geuploadet wurde.');
 define('lg_home_admin_toomanyusers','Es sind mehr aktive User registriert als die Lizenz zulässt. Bitte User deaktivieren oder die Lizenz anpassen.');
 define('lg_home_admin_customerid','Kunden ID');
-define('lg_home_admin_licusers','Staff/Lizenzen');
-define('lg_home_admin_licsupport','Support läuft aus');
+define('lg_home_admin_licusers', 'Staff/%s Licenses');
```

Comment:
@Josefism I think here we can do Staff/%s Lizenzen

Reply:
@mpstenson Done and repushed.